content/millennium/r4/scheduling/appointment.md

```diff
@@ -45,6 +45,7 @@ The following fields are returned if valued:
 * [End date time](http://hl7.org/fhir/DSTU2/appointment-definitions.html#Appointment.end){:target="_blank"}
 * [Duration in minutes](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.minutesDuration){:target="_blank"}
 * [Comment](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.comment){:target="_blank"}
+* [Patient Instruction](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.patientInstruction){:target="_blank"}
```

Comment:
Nit: we can update this to HTTPS but I am okay with leaving it as it is as well.

Reply:
I left it as http as the others also are